datafusion/common/src/config.rs

```diff
@@ -1552,7 +1552,7 @@ config_namespace_with_hashmap! {
     /// Options controlling parquet format for individual columns.
     ///
     /// See [`ParquetOptions`] for more details
-    pub struct ColumnOptions {
+    pub struct ParquetColumnOptions {
```

Comment:
We could additionally (later) move the Parquet functionality to Parquet-related modules/files.
Perhaps just for organizational purposes.